src/test/java/api/loans/DueDateScheduledNoticesProcessingTests.java

```diff
@@ -105,8 +101,8 @@ public void beforeEach() {
   public void beforeNoticeShouldBeSentAndItsNextRunTimeShouldBeUpdated() {
 
     DateTime beforeDueDateTime = dueDate.minus(beforePeriod.timePeriod()).plusSeconds(1);
+    templateFixture.createDummyNoticeTemplate(beforeTemplateId);
```

Comment:
What is the `beforeTemplateId`?

Reply:
This test already used  `beforeTemlateId` constant. I only added it to templateFixture to avoid test fail.

Reply:
I understand that, what does that constant represent?

Reply:
Ok, I find the name confusing. I will resolve this comment.

Reply:
It represents the ID of a template used for courtesy patron notices (the ones sent **before** upcoming due date)